modules/relax/layouts/relax.cfm

```diff
@@ -16,8 +16,6 @@
     <title>ColdBox Relax - RESTful Tools For Lazy Experts</title>
     <!--- favicon --->
     <link href="#prc.root#/includes/images/favicon.ico" rel="shortcut icon" type="image/x-icon" />
-    <!--- SES --->
```

Comment:
@evagoras If this is removed, then all relative resources will fail to load though.